src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -2411,14 +2411,6 @@ protected void sendChunkUpdates(Chunk newChunk) {
         return super.teleport(position, chunks, flags);
     }
 
-    /**
-     * Represents the main or off hand of the player.
-     */
-    public enum Hand {
```

Comment:
I think this should stay on Player. Even if it is slightly incorrect (eg for LivingEntity#swingHand to take a Player.Hand), I believe most people will look for player hand. Also makes this change less breaking

Reply:
I don't know if it's that breaking, using `Player.Hand` is still correct. Do you still want this enum to be in a player class?